backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/LayoutService.java

```diff
@@ -92,9 +99,29 @@ public Diagram layout(Diagram diagram) {
         Map<String, ElkGraphElement> id2ElkGraphElements = convertedDiagram.getId2ElkGraphElements();
         Diagram layoutedDiagram = this.elkLayoutedDiagramProvider.getLayoutedDiagram(diagram, elkDiagram, id2ElkGraphElements);
 
+        if (this.elkGraphJsonIsEnable()) {
```

Comment:
`if (this.logger.isDebugEnabled())`

Reply:
I would put a flag there to prevent the display of the ELK graph every time we create a diagram representation, like an option in _application-dev.properties_.

Reply:
You do not need that since SLF4J already provides that, truth to be told we should probably remove the existing setting in `application-dev.properties` to let each developer choose what they want.